facades/PC/src/main/java/org/terasology/engine/Terasology.java

```diff
@@ -426,4 +457,24 @@ private static GameManifest getLatestGameManifest() {
         return latestGame.getManifest();
     }
 
+    private static int getPositionOfLastDigit(String str) {
+        int position;
+        for (position = str.length() - 1; position >= 0; --position) {
+            char c = str.charAt(position);
+            if (!Character.isDigit(c)) {
+                break;
+            }
+        }
+        return position + 1;
+    }
+
+    private static int getLastNumber(String str) {
+        int positionOfLastDigit = getPositionOfLastDigit(str);
+        if (positionOfLastDigit == str.length()) {
+            // string does not end in digits
+            return -1;
+        }
+        return Integer.parseInt(str.substring(positionOfLastDigit));
+    }
+
```

Comment:
these last two methods help find the last integer at the end of the title of the last game.